android/src/main/java/com/rudderstack/react/android/RNRudderSdkModule.java

```diff
@@ -147,6 +147,32 @@ public void identify(String userId, ReadableMap traits, ReadableMap options) {
         rudderClient.identify(userId, Utility.convertReadableMapToTraits(traits), Utility.convertReadableMapToOptions(options));
     }
 
+    @ReactMethod
+    public void alias(String newId, ReadableMap options) {
+        if (rudderClient == null) {
+            RudderLogger.logWarn("Dropping the Alias call as RudderClient is not initialized yet, Please use `await` keyword with the setup call");
+            return;
+        }
+        if (TextUtils.isEmpty(newId)) {
+            rudderClient.alias(Utility.convertReadableMapToOptions(options));
```

Comment:
We are not having an alias(options) api in Android, can you please change it to use a valid alias API ?

Reply:
We do have [alias(option)](https://github.com/rudderlabs/rudder-sdk-android/blob/10ff3da5112979f1c1f82115ecf35d6665e9a319/core/src/main/java/com/rudderstack/android/sdk/core/RudderClient.java#L500) API. Right?

Reply:
Yeah, we had alias(newId, option) API, but not alias(option).

Reply:
Oh, okay.